components/Ink/tests/unit/MDCInkViewTests.m

```diff
@@ -135,4 +135,16 @@ - (void)testTraitCollectionDidChangeBlockCalledWithExpectedParameters {
   XCTAssertEqual(passedTraitCollection, fakeTraitCollection);
 }
 
+- (void)testNullResettableInkColor {
+  // Given
+  MDCInkView *testInkView = [[MDCInkView alloc] init];
+  testInkView.inkColor = UIColor.redColor;
+
+  // When
+  testInkView.inkColor = nil;
+
+  // Then
+  XCTAssertEqual(testInkView.inkColor, testInkView.defaultInkColor);
```

Comment:
Tip: Capture the default value of `inkColor` in the "Given" section then compare the two colors in the "Then" rather than relying on the `defaultInkColor` API.